examples/data/e2e_tests/README.md

```diff
@@ -0,0 +1,78 @@
+                    Copyright 2021 ABSA Group Limited
+                  
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+                You may obtain a copy of the License at
+               http://www.apache.org/licenses/LICENSE-2.0
+            
+     Unless required by applicable law or agreed to in writing, software
+       distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+                      limitations under the License.
+
+# Examples of Enceladus E2E data
+
+Main purpose of the data presented here is for use in Enceldus e2e automated tests,
+but it also nicely shows examples of data and combinations possible.
+
+There are 3 folders
+
+- `data` - data for hdfs and jsons ready to be imported into Menas
+- `setup_scripts` - should help you with importing data into Menas and copying onto HDFS
+- `test_jsons` - jsons ready to be executed using [Hermes](https://github.com/AbsaOSS/hermes)
+
+## Setup Scripts
+
+### Import into Menas
+
+**Tools used**: cURL
+
+This script will populate Menas with datasets, schemas and mapping tables.
+To run scripts with user prompts, run:
+
+```shell
+./setup_scripts/import_all.sh
+```
+
+Ths will start the script and ask you for a username, password and Menas URL.
+
+To run without the prompts use:
+```shell
+./setup_scripts/import_all.sh with user changeme http://localhost:8080/menas
```

Comment:
maybe add an explanatory comment, otherwise, it looks like _using a username 'changeme'_